src/test/java/com/codersbay/gerhofer/DummyClassTest.java

```diff
@@ -24,4 +26,58 @@ public void testDoNotCallThrowsException() {
         });
     }
 
+
+
+    @Test
+    public void rectangleTest() {
+        rectangle r1 = new rectangle(1,2);
+        assertEquals(r1.getArea(), r1.width * r1.length);
```

Comment:
in tests ists immer cool wenn man nicht die rechnung nochmal hinschreibt (so kopiert man ja die logik) 

sondern wenn man konkrete fälle hat 

also hier wärs besser du würdest schreibn: 
assertEquals(r1.getArea(), 2)
assertEquals(r1.getPerimeter(), 4)

etc